test/splitText.test.js

```diff
@@ -0,0 +1,50 @@
+import { expect, test } from 'vitest'
```

Comment:
Thanks for adding the test!
Although you might want to add test cases that covers the case where chunk size is a bit larger and we pack multiple sentences in one chunk.